packages/pigeon/lib/swift_generator.dart

```diff
@@ -43,6 +43,44 @@ class SwiftOptions {
 /// Calculates the name of the codec that will be generated for [api].
 String _getCodecName(Api api) => '${api.name}Codec';
 
+/// Returns a new [Method] using the swift function signature from [func],
+/// ie the name of function e the strings between the semicolons.
+/// Example:
```

Comment:
Nit: blank line above this so the first sentence is its own paragraph, per Dart style.

Reply:
Fixed